lib/hterm.js

```diff
@@ -102,13 +102,19 @@ hterm.Keyboard.prototype.onKeyDown_ = function (e) {
   }
 
   if (e.metaKey || e.altKey) {
+    if (e.key !== 'Alt' && e.altKey && global.config.getConfig().pass_meta_keys) {
```

Comment:
Using `config.getConfig()` skips the decoration process (so you won't get the config processed by plugins). The `getConfig` wrapper in `lib/utils/config` fixes this. Normally config values are loaded into the UI reducer though, instead of accessed directly like here. I'm not sure if that's necessary here though, since you'll do the check on every alt button press (and thus won't need the config to reload or similar).